Source/Swig/swig.h

```diff
@@ -335,7 +335,10 @@ extern int        ParmList_is_compactdefargs(ParmList *p);
   extern void Swig_offset_string(String *s, int number);
   extern String *Swig_pcre_version(void);
   extern void Swig_init(void);
+  extern void Swig_warn(const char *filename, int line, const char *msg);
+  
```

Comment:
It's actually a (very) old function and not a new one and was removed by 1e8fa8fee022de3151209d0719aa30aa0fdb4e65 from master but left in this branch. Removed too now.